server/block/ice.go

```diff
@@ -0,0 +1,73 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/block/model"
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/df-mc/dragonfly/server/world/particle"
+	"math/rand"
+)
+
+// Ice is a transparent block that forms when water freezes and melts when it is near a bright light source.
+type Ice struct {
+	solid
+}
+
+// LightDiffusionLevel ...
+func (Ice) LightDiffusionLevel() uint8 {
+	return 3
+}
+
+// BreakInfo ...
+func (i Ice) BreakInfo() BreakInfo {
+	b := newBreakInfo(0.5, alwaysHarvestable, pickaxeEffective, silkTouchOnlyDrop(i))
+	b.withBreakHandler(func(pos cube.Pos, w *world.World, u item.User) {
+		if w.Dimension().WaterEvaporates() {
+			w.AddParticle(pos.Vec3Centre(), particle.Evaporate{})
+			return
+		}
+		if p, ok := u.(interface {
+			GameMode() world.GameMode
+		}); ok && p.GameMode().CreativeInventory() {
+			return
+		}
+		if mainHand, _ := u.HeldItems(); hasSilkTouch(mainHand.Enchantments()) {
+			return
+		}
+		if _, ok := w.Block(pos.Side(cube.FaceDown)).Model().(model.Solid); !ok {
+			return
+		}
+		w.SetBlock(pos, Water{}, nil)
+	})
+	b.withBlastResistance(0.5)
```

Comment:
```suggestion
```
https://github.com/df-mc/dragonfly/blob/master/server/block/break_info.go#L102

Reply:
0.5 * 5 = 2.5, which is not 0.5 as stated by the wiki

Reply:
The wiki values are 1/5 of the values we used within dragonfly.

Reply:
Alright then